proof/refine/RISCV64/Ipc_R.thy

```diff
@@ -5343,6 +5350,18 @@ crunch ifCondRefillUnblockCheck
   for reply_projs[wp]: "\<lambda>s. P (replyNexts_of s) (replyPrevs_of s) (replyTCBs_of s) (replySCs_of s)"
   (wp: crunch_wps simp: crunch_simps)
 
+lemma replyCanGrant_update_valid_replies'_sc_asrt[wp]:
+  shows "updateReply ptr (replyCanGrant_update f) \<lbrace>valid_replies'_sc_asrt ptr\<rbrace>"
+  apply (rule valid_replies'_sc_asrt_lift; wpsimp wp: updateReply_wp_all)
+    apply (subst opt_map_upd_triv)
+     apply (clarsimp simp: opt_map_def obj_at'_def)
+    apply clarsimp
+   apply (subst opt_map_upd_triv)
```

Comment:
both here and in the previous case, wouldn't `subst opt_map_upd_triv; clarsimp ...` be shorter?